src/qtAliceVision/SingleImageLoader.cpp

```diff
@@ -90,6 +90,16 @@ void SingleImageLoadingIORunnable::run()
     }
     catch (const std::runtime_error& e)
     {
+        // std::runtime_error at this point is a "can't find/open image" error
+        if (access(_reqData.path.c_str(), F_OK) == -1)  // "can't find image" case
```

Comment:
std::filesystem::exists

Reply:
in other places, we currently use boost::filesystem::exists as it was before c++17.

Reply:
Done.